<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBPrintDialogUi</class>
 <widget class="QDialog" name="QBPrintDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Preview</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="descrLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox1">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="Q3TextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
       <property name="shortcut">
        <string>F1, Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="setupButton">
       <property name="text">
        <string>Page Setup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fontButton">
       <property name="text">
        <string>Select Font</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>Q3TextEdit</extends>
   <header>Qt3Support/Q3TextBrowser</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textBrowser</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>printButton</tabstop>
  <tabstop>setupButton</tabstop>
  <tabstop>fontButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>abortButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
